Datadog.Trace.sln

```diff
@@ -433,6 +433,10 @@ Project("{9A19103F-16F7-4668-BE54-9A1E7A4F7556}") = "DuplicateTypeProxy", "test\
 EndProject
 Project("{FAE04EC0-301F-11D3-BF4B-00C04F79EFBC}") = "Samples.AspNetCoreRazorPages", "test\test-applications\integrations\Samples.AspNetCoreRazorPages\Samples.AspNetCoreRazorPages.csproj", "{1B9E6BF4-9D48-4988-9945-248096119E46}"
 EndProject
+Project("{FAE04EC0-301F-11D3-BF4B-00C04F79EFBC}") = "Datadog.Trace.Tools.Analyzers", "src\Datadog.Trace.Tools.Analyzers\Datadog.Trace.Tools.Analyzers.csproj", "{5450EA0B-56D3-4E29-932E-094AD037B345}"
```

Comment:
This project should probably live in the top-level `tools` folder.

Reply:
I specifically didn't put it in there because it needs to be built with all the _src_ projects, which means fewer updates to the CI. But as I can't get the `x86` build to pass-as-is this is probably going to be shelved until I've rehashed the CI anyway, so I'll look into it then